src/ForneyLab.jl

```diff
@@ -5,13 +5,15 @@ using Base64: base64encode
 using LinearAlgebra: diag, det, tr, cholesky, pinv, PosDefException
 using SparseArrays: spzeros
 using SpecialFunctions: digamma, erfc, logfactorial, logabsgamma, logabsbeta, gamma, loggamma, erf
-using LinearAlgebra: Diagonal, Hermitian, isposdef, ishermitian, I, tr
+using LinearAlgebra: Diagonal, Hermitian, isposdef, ishermitian, I, tr, eigvals, logdet
 using InteractiveUtils: subtypes
 using Printf: @sprintf
 using StatsFuns: logmvgamma, betainvcdf, gammainvcdf, poisinvcdf
 using ForwardDiff
 using StatsBase: Weights
 using DataStructures: Queue, enqueue!, dequeue!
+using Flux.Optimise
+using Zygote
```

Comment:
Any chance we can avoid `using Zygote`?
It will affect the compilation speed of ForneyLab.

Reply:
@albertpod @semihakbayrak https://github.com/JuliaPackaging/Requires.jl

Reply:
This package allows you to extend package's functionality if other packages (e.g. `Zygote`) are loaded already and at the same time the original package does not depend on `Zygote`.